plugin/src/main/java/io/snyk/languageserver/protocolextension/messageObjects/SnykScanParam.java

```diff
@@ -0,0 +1,63 @@
+package io.snyk.languageserver.protocolextension.messageObjects;
+
+import java.util.Objects;
+
+public class SnykScanParam {
```

Comment:
would be nice for this to be a record, too, but it may be problematic with deserialization with GSon, even though officially 2.10 supports records.

Reply:
thing is we don't reference it directly. It would be nice to replace jackson with it